concourse/model/traits/update_component_deps.py

```diff
@@ -65,6 +71,15 @@ class UpstreamUpdatePolicy(enum.Enum):
         type=MergePolicy,
         doc='whether or not created PRs should be automatically merged',
     ),
+    AttributeSpec.optional(
+        name='merge_method',
+        default='merge',
+        type=MergeMethod,
+        doc=(
+            'The method to use when merging PRs automatically. The default is `merge`, i.e. to '
+            'create a merge commit.'
```

Comment:
nit: might link to GitHub's documentation (https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github)

Reply:
👍